spec/fixtures/installed_no_logging.js

```diff
@@ -0,0 +1,80 @@
+(function() {
+  var self = this;
+  var reporter = window.ZendeskReporter ? new window.ZendeskReporter() : {};
+  function wrapped$() {
+    reporter.increment && reporter.increment(
+      'app_framework.app_scope.violated',
+      1,
+      ['origin:' + self.location.hostname]
+    );
+    return self.$.apply(null, arguments);
+  }
+
+  var wrappedWindow = this.$.extend(true, {}, self, { $: wrapped$ });
+  wrappedWindow.window = wrappedWindow;
+  wrappedWindow.top = wrappedWindow;
+
+  with( ZendeskApps.AppScope.create() ) {
+    require.modules = {
+        "a.js": function(exports, require, module) {
+          var a = {
+    name: 'This is A'
+  };
```

Comment:
The alignment in this file looks broken, same for other fixture files.

Reply:
I've just copy pasted it from another fixture `legacy_app_en.js`, and the broken alignment of the other code actually matches the way `installed.js` looks in production.